go/vt/vttablet/tabletserver/planbuilder/plan.go

```diff
@@ -162,9 +162,6 @@ type Plan struct {
 	// Permissions stores the permissions for the tables accessed in the query.
 	Permissions []Permission
 
-	// FieldQuery is used to fetch field info
-	FieldQuery *sqlparser.ParsedQuery
-
```

Comment:
When I tried to `make codegen`, I got the following errors:
```
% make codegen
2022/06/08 18:56:21 saved '/Users/arvind/go/src/vitess.io/vitess/go/vt/sqlparser/ast_clone.go'
2022/06/08 18:56:21 saved '/Users/arvind/go/src/vitess.io/vitess/go/vt/sqlparser/ast_visit.go'
2022/06/08 18:56:21 saved '/Users/arvind/go/src/vitess.io/vitess/go/vt/sqlparser/ast_rewrite.go'
2022/06/08 18:56:21 saved '/Users/arvind/go/src/vitess.io/vitess/go/vt/sqlparser/ast_equals.go'
2022/06/08 18:56:31 /Users/arvind/go/src/vitess.io/vitess/go/vt/vttablet/tabletserver/planbuilder/cached_size.go:63:17: cached.FieldQuery undefined (type *Plan has no field or method FieldQuery)
2022/06/08 18:56:31 /Users/arvind/go/src/vitess.io/vitess/go/vt/vttablet/tabletserver/cached_size.go:36:50: cached.Fields undefined (type *TabletPlan has no field or method Fields)
2022/06/08 18:56:31 /Users/arvind/go/src/vitess.io/vitess/go/vt/vttablet/tabletserver/cached_size.go:37:31: cached.Fields undefined (type *TabletPlan has no field or method Fields)
2022/06/08 18:56:31 failed to load packages
exit status 1
make: *** [sizegen] Error 1
```

I had to manually edit `cached_size.go` and remove the deleted `FieldQuery` and then run `make codegen` again.
This seems like a bug in `make sizegen`.

Reply:
cc @vmg

Reply:
This is the limitation of generated files in go (or at least in how we do generated files).
This is not a bug for now.

May be @vmg can add details better.